lib/Configuration.js

```diff
@@ -36,6 +36,10 @@ const DEFAULT_CONFIG = {
   stripFromPath: null,
   tab: '  ',
   useRelativePaths: true,
+  packageDependencies: ({ config }: Object): Array<string> =>
```

Comment:
It might be nice to be more specific about the shape of this Object with this type declaration. We could likely do this better in a number of places.